### ISSUE TITLE
Installation via pip fails

### ISSUE THREAD

**Issue by novel** [johnthagen/cpplint-junit#2]
```
vagrant@ubuntu-bionic:~$ sudo -H pip install cpplint-junit
Collecting cpplint-junit
  Downloading https://files.pythonhosted.org/packages/73/99/8910702390599b512b5204af36b75100e8ab12485787346d17d306f1696f/cpplint-junit-1.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-ZD4gVK/cpplint-junit/setup.py", line 18, in <module>
        install_requires=open('requirements.txt').readlines(),
    IOError: [Errno 2] No such file or directory: 'requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ZD4gVK/cpplint-junit/
You are using pip version 9.0.1, however version 19.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
vagrant@ubuntu-bionic:~$ 
```

**Comment by johnthagen** [comment]
@novel I'll fix this, but just FYI, you might find https://pypi.org/project/cpplint/ a better tool. It supports JUnit output natively.

**Comment by johnthagen** [comment]
Fixed in 0fac1a5e98e19cd3c37e7e09d41c4ed8cd8a494c (v1.2.1)

**Comment by novel** [comment]
Thanks for the fix and for suggestion on the cpplint fork.
